### PR TITLE
Update aws-lambda.md

### PR DESCRIPTION
**Pull request by lafernando**
$subject.

### CODE REVIEW

File: learn/deployment/aws-lambda.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ redirect_from:
 
 # AWS Lambda
 
-The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function. This is done by importing the `ballerinax/awslambda` module and simply annotating the Ballerina function with the `awslambda:Function` annotation. Also, the Ballerina function must have the following signature: `function (awslambda:Context, json) returns json|error`. 
+The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function. This is done by importing the `ballerinax/awslambda` module and simply annotating the Ballerina function with the `awslambda:Function` annotation. Also, the Ballerina function must have the following signature: `function (awslambda:Context, json|EventType) returns json|error`. The second parameter in the function is either a `json` or a domain specific event type such as `awslambda:S3Event`. If the user does not provide a domain specific event type, the JSON representation of the incoming event will be sent to the function.
```
Comment (anupama-pathirage) [inline learn/deployment/aws-lambda.md:14]:
```suggestion
The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function. This is done by importing the `ballerinax/awslambda` module and simply annotating the Ballerina function with the `awslambda:Function` annotation. Also, the Ballerina function must have the following signature: `function (awslambda:Context, json|EventType) returns json|error`. The second parameter in the function is either a `json` or a domain-specific event type such as `awslambda:S3Event`. If the user does not provide a domain specific event type, the JSON representation of the incoming event will be sent to the function. 
```

Comment (praneesha) [inline learn/deployment/aws-lambda.md:14]: Shall we break this into two paragraphs? 
```suggestion
The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function. This is done by importing the `ballerinax/awslambda` module and simply annotating the Ballerina function with the `awslambda:Function` annotation. Also, the Ballerina function must have the following signature: `function (awslambda:Context, json|EventType) returns json|error`. The second parameter in the function is either a `json` or a domain-specific event type such as `awslambda:S3Event`. If the user does not provide a domain specific event type, the JSON representation of the incoming event will be sent to the function. 
```

File: learn/deployment/aws-lambda.md [hunk 2]
```diff
@@ -23,9 +23,34 @@ import ballerina/crypto;
...
+The first parameter with the [awslambda:Context](/learn/api-docs/ballerina/awslambda/objects/Context.html) object contains the information and operations related to the current function execution in AWS Lambda such as the request ID and the remaining execution time. The second parameter contains the input request data. This input value will vary depending on the source, which invoked the function (e.g., an AWS S3 bucket update event). The return type of the function is `json|error`, which means in a successful scenario, the function can return a `json` value with the response or else in an error situation, the function will return an `error` value, which provides information on the error to the system. You can also provide functions, which do not return anything at all, which implicitly signals a successful execution without a returning result.
```
Comment (praneesha) [inline learn/deployment/aws-lambda.md:53]: Better if we break this also to 2 paragraphs.

File: learn/deployment/aws-lambda.md (hunk 1 above)
Comment (praneesha) [inline learn/deployment/aws-lambda.md:14]: Shall we break this into two paragraphs?

File: learn/deployment/aws-lambda.md (hunk 2 above)
Comment (lafernando) [reply learn/deployment/aws-lambda.md]: +1, done.

File: learn/deployment/aws-lambda.md (hunk 1 above)
Comment (lafernando) [reply learn/deployment/aws-lambda.md]: +1, done.

**Review by praneesha — APPROVED** [review]

**Pull request merged**